optaplanner-core/src/main/java/org/optaplanner/core/api/solver/SolverManager.java

```diff
@@ -51,6 +51,22 @@
     // Static creation methods: SolverConfig and SolverFactory
     // ************************************************************************
 
+    /**
+     * Use a {@link SolverConfig} to build a {@link SolverManager}.
+     * <p>
+     * When using {@link ScoreManager} too, use {@link #create(SolverFactory)} instead
+     * so they reuse the same {@link SolverFactory} instance.
+     *
+     * @param solverConfig never null
+     * @return never null
+     * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
+     * @param <ProblemId_> the ID type of a submitted problem, such as {@link Long} or {@link UUID}.
```

Comment:
```suggestion
     * @param <ProblemId_> the ID type of a submitted problem, such as {@link Long} or {@link UUID}
```

As much as I don't like it, that's the convention.